README.md

```
@@ -1,58 +1,319 @@
+       "SmtpPort": your_smtp_port,
+       "SmtpUsername": "your_smtp_username",
+       "SmtpPassword": "your_smtp_password"
+     },
+     "JwtSettings": {
+       "SecretKey": "your_jwt_secret_key"
+     },
+     "QRCodeSecretKey": "your_qr_code_secret_key"
+   }
+   ```
+
+3. **Database Setup:** 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)